### PR TITLE
must run tests

### PR DESCRIPTION
**Pull request by MHBauer**
don't want to accidentally not run the tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -231,6 +231,7 @@ coverage: .init
 	$(DOCKER_CMD) contrib/hack/coverage.sh --html "$(COVERAGE)" \
 	  $(addprefix ./,$(TEST_DIRS))
 
+.PHONY: test test-unit test-integration test-e2e
```
Comment (duglin) [inline Makefile]: I think this is supposed to be "test-dep" instead of "test-e2e"

Comment (jboyd01) [reply Makefile]: test-dep is already called out as phony on line 384 (correctly now with Moran's fix).  I think it's correct to add test-e2e as well.

File: Makefile [hunk 2]
```diff
@@ -379,7 +380,7 @@ $(BINDIR)/svcat: .init .generate_files cmd/svcat/main.go
 	$(DOCKER_CMD) $(GO_BUILD) -o $@ $(SC_PKG)/cmd/svcat
 
 # Dependency management via dep (https://golang.github.io/dep)
-PHONHY: verify-vendor test-dep
+.PHONY: verify-vendor test-dep
```
Comment (carolynvs) [inline Makefile]: Doh! Thank you for fixing. 😊

**Comment by duglin** [comment]
LGTM

**Pull request merged**